src/components/islets/search-modal/index.tsx

```diff
@@ -15,7 +15,7 @@ import {
 import { Input } from "@/components/ui/input";
 
 export default function SearchModal() {
-  const [open, setOpen] = useState(false);
+  const [open, setOpen] = useState<boolean>(false);
```

Comment:
@MikeMenos what's the purpose of setting boolean type as typescript will automatically infer boolean?

Reply:
I know but in some other useState cases that I saw, like ` const [active, setActive] = useState<string>("default");`, you've explicitly added the type so I recommended it for consistency purposes.